onboard/nav/stateMachine.hpp

```diff
@@ -112,6 +116,12 @@ class StateMachine
     // Bool of whether radio repeater has been dropped.
     bool mRepeaterDropComplete = false;
 
+    // Bool of whether it is time to drop repeater.
+    bool mIsTimeToDropRepeater = false;
```

Comment:
Initialize in constructor not in .hpp file. This way we only have to look in one place when we are looking for bugs and suspect something is initialized.